### PR TITLE
multiple enhancements

### PR DESCRIPTION
**Pull request by achubaty**
* try to automatically detect python path when setting tiler option (#19)
* fixed a race condition in temp files running `tile()` in parallel (with #18)
* use GitHub Actions for testing (#6)
* added myself (@achubaty) as contributor (per https://github.com/ropensci/tiler/issues/6#issuecomment-1271886041)
* updated NEWS
* version bump to 0.2.6

### CODE REVIEW

File: tiler.Rproj [hunk 1]
```diff
@@ -18,3 +18,7 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace
```
Comment (leonawicz) [inline tiler.Rproj:21]: Just curious what is the motivation for the new lines in this file.

**Review by leonawicz — APPROVED** [review]
I will merge this. I will have some additional changes to follow shortly.

File: tiler.Rproj (hunk 1 above)
Comment (achubaty) [reply tiler.Rproj:21]: That's just me setting Rstudio project options to auto-generate package docs etc.

**Comment by achubaty** [comment]
Thank you!

File: tiler.Rproj (hunk 1 above)
Comment (leonawicz) [reply tiler.Rproj:21]: Ah, thanks!

**Comment by leonawicz** [comment]
@achubaty I have pushed my changes for now. For context, the machine I am currently testing on is Windows with OSGeo4W installed and in use. Feel free to continue with any outstanding issues or improvements in a new PR from here.

Thank you for your assistance! :)

I will update documentation and other package metadata later when the package is ready for a new release.

**Pull request merged**